Chapter-4-applying-tactical-domain-driven-design/Docs/ArchitectureDecisionLog/0024-contracts-error-handling.adoc

```diff
@@ -0,0 +1,46 @@
+= 24. Error Handling in Contracts Module
+
+Date: 2024-07-23
+
+== Problem
+
+On current project state Contracts module bussiness logic is getting more complex and it is hard to control flow of errors in the module. 
+
+The current error handling approach is based on exceptions, which makes it difficult to follow the appliation flow and maintain the codebase. The team is considering adopting a new approach to handle errors in the Contracts module.
+
+== Options
```

Comment:
Options can be a part of Problem (===). This way we do not change the structure of ADRs, therefore no one will be mislead by completely different ADR structure in comparison to others

Reply:
Correct :D I will change it.